services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py

```diff
@@ -302,15 +302,24 @@ async def get_study_job_outputs(
     return job_outputs
 
 
-@router.post(
-    "/{study_id}/jobs/{job_id}/outputs/logfile",
-    response_class=RedirectResponse,
-    include_in_schema=API_SERVER_DEV_FEATURES_ENABLED,
-    status_code=status.HTTP_501_NOT_IMPLEMENTED,
+@router.get(
+    "/{study_id}/jobs/{job_id}/outputs/log-links",
```

Comment:
THOUGHT: perhaps we should also have also the same in the solver jobs?

Reply:
In the solver jobs case we just use a redirect. I don't see any reason to change that. Only for cosmetics, so that the solver jobs api is as identical to the study jobs api as possible. But I think we have bigger fish to catch.